README.md

```diff
@@ -63,29 +63,37 @@ species,file,taxonomy,shallow,broad,mode,uniprot,mcl_test
 
 4. Create a parameter file that includes all necessary input, output, and parameter specifications - example below:
 ```
 {
-  "input": "/home/ubuntu/environment/github/phylorthology/test/complete_samplesheet.csv",
-  "mcl_test_input": "/home/ubuntu/environment/github/phylorthology/test/mcl_test_samplesheet.csv",
-  "s3_dir": "s3://organism-selection-engines/sequence-data/aa-seqs/phylorthology-test",
-  "outdir": "results-s3-test",
-  "mcl_inflation": [ "0.8", "0.9", "1.0", "1.1", "1.2", "1.3", "1.4", "1.5", "1.6", "1.7" ]
+  "input": "<FILE_PATH_TO_YOUR_SAMPLESHEET>",
+  "outdir": "results",
+  "mcl_inflation": "0.8,0.9,1.0,1.1,1.2,1.3,1.4,1.5,1.6,1.7"
 }
 ```
 
-5. Ensure that proteins are named following the following convention. 
+Alternatively, you can use the test dataset provided by Arcadia Science [here](https://github.com/Arcadia-Science/test-datasets/phylorthology).
+
+5. Ensure that proteins are named following the following convention.
 - Species_genus:GeneID
-- Example: 
+- Example:
 `>Entamoeba_histolytica:C4M6M9 tr|C4M6M9|C4M6M9_ENTHI NGG1-interacting factor 3, putative OS=Entamoeba histolytica HM-1:IMSS OX=294381 GN=EHI_161860 PE=3 SV=1`
-- Everything prior to the colon (:) is a constant (and unique to that species/lineage) identifier and can be whatever you would like, but must not include a colon. Everything that follows the colon is what must be a protein/gene identifier unique to that sequence. Additional sequence info may be included following a space. If the proteome comes from UniProt, this must be the UniProt protein accession. We use the string that follows the colon to extract the uniprot accession and annotate proteins. 
+- Everything prior to the colon (:) is a constant (and unique to that species/lineage) identifier and can be whatever you would like, but must not include a colon. Everything that follows the colon is what must be a protein/gene identifier unique to that sequence. Additional sequence info may be included following a space. If the proteome comes from UniProt, this must be the UniProt protein accession. We use the string that follows the colon to extract the uniprot accession and annotate proteins.
+
+6. Download the pipeline and test it on a minimal dataset with a single command run in the root of this repository:
+
+   ```bash
+   # If you're using your own test data
+   nextflow run . -profile docker -params-file parameters.json
+   ```
 
-6. Download the pipeline and test it on a minimal dataset with a single command run in the "test" directory of this repository:
+   OR
 
    ```bash
-   nextflow run phylorthology -profile docker -params-file parameters.json
+   # If you're using Arcadia Science's test dataset
+   nextflow run . -profile docker -params-file https://github.com/Arcadia-Science/test-datasets/raw/main/phylorthology/nextflow_parameters.json
```

Comment:
Oh, interesting - so you can specify the path to a remote parameter file as well? I imagine this would work the same if you stored it on s3, for instance in the same bucket where the test data is located? (using the s3 path of course)

Reply:
Yeah I think it should work with an S3 path